docs/getting-started/publishing-data.md

```diff
@@ -19,19 +19,25 @@ You can read and learn about CSV in [this post](/docs/data-packages/csv).
 
 ## Get some CSV data
 
-You'll need a CSV file to push.
-
 Let's have a simple CSV file. Create a file in your current directory named `mydata.csv` and paste the following into it (you can use your own):
 
+**Note:** *It will validate your data prior publishing, if everything is fine - otherwise you'd get validation errors*
```

Comment:
Think something like this would be better:
We are checking the validity of your data prior publishing. Find out more about data validation and datasets here: some link...

Reply:
prior *to* publishing

`prior publishing` is not grammatical english. (Are you guys checking this stuff in a grammar checker?)

Reply:
This is really good idea, but where I need to link it to http://datahub.io/docs/data-packages/csv. We do not have docs for data validation